src/main/groovy/com/ullink/SourceLinkIndexing.groovy

```diff
@@ -42,6 +42,10 @@ class SourceLinkIndexing extends ConventionTask {
         }
     }
 
+    def findRepoRootDir(File dir) {
+        new File((File) dir, '.git').exists() ? dir : findRepoRootDir(dir.getParentFile())
```

Comment:
It didn't know which constructor to take new File(String, String) or new File(File, String). I'll check again.

I'll do it for the limit.